src/pages/publication/references_tab/sub_type_forms/JournalArticleForm.tsx

```diff
@@ -0,0 +1,33 @@
+import { FormikProps, useFormikContext } from 'formik';
+import React, { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import { JournalPublication } from '../../../../types/publication.types';
+import { JournalEntityDescription } from '../../../../types/publication_types/journalPublication.types';
+import { ReferenceFieldNames } from '../../../../types/publicationFieldNames';
+import PeerReview from '../components/PeerReview';
+import NviValidation from '../components/NviValidation';
+import JournalForm from './JournalForm';
+
+const JournalArticleForm: FC = () => {
+  const { t } = useTranslation('publication');
+  const { values }: FormikProps<JournalPublication> = useFormikContext();
+  const {
+    reference: { publicationContext, publicationInstance },
+  } = values.entityDescription as JournalEntityDescription;
+
+  return (
+    <>
+      <JournalForm />
+
+      <PeerReview fieldName={ReferenceFieldNames.PEER_REVIEW} label={t('references.peer_review')} />
+
+      <NviValidation
```

Comment:
vet vi når dette skal vises? i dag viser vi det vel for alle typer, også hvor det aldri vil kunne inngå i NVI

Reply:
hm, jeg bare antok at det ikke skulle vises hvis man ikke hadde mulighet til å sette `isPeerReviewed`, men kanskje det skal være med allikevel? Jeg kan sjekke featurebeskrivelsene igjen

Reply:
kan ikke huske å ha sett det i noen feature før, men fra før viser vi det alltid

Reply:
ok, da endrer jeg tilbake til å vise NviValidation på alle